### PR TITLE
Always log_info when we broadcast a transaction, including the txid

### PR DESCRIPTION
**Pull request by TheBlueMatt**

### CODE REVIEW

File: lightning/src/ln/onchaintx.rs [hunk 1]
```diff
@@ -926,6 +926,7 @@ impl<ChannelSigner: Sign> OnchainTxHandler<ChannelSigner> {
 			if let Some((new_timer, new_feerate, bump_tx)) = self.generate_claim_tx(height, &claim_material, &&*fee_estimator, &&*logger) {
 				claim_material.height_timer = new_timer;
 				claim_material.feerate_previous = new_feerate;
+				log_info!(logger, "Broadcasting onchaing {}", log_tx!(bump_tx));
```
Comment (valentinewallace) [inline lightning/src/ln/onchaintx.rs:929]: nit: onchain

File: lightning/src/chain/channelmonitor.rs [hunk 2]
```diff
@@ -1568,6 +1568,7 @@ impl<Signer: Sign> ChannelMonitorImpl<Signer> {
 					L::Target: Logger,
 	{
 		for tx in self.get_latest_holder_commitment_txn(logger).iter() {
+			log_info!(logger, "Broadcasting local {}", log_tx!(tx));
```
Comment (valentinewallace) [inline lightning/src/chain/channelmonitor.rs:1571]: nit: could say "local commitment transaction" for clarity

**Review by valentinewallace — APPROVED** [review]
LGTM, just some nits

**Review by jkczyz — APPROVED** [review]

File: lightning/src/chain/channelmonitor.rs (hunk 2 above)
Comment (TheBlueMatt) [reply lightning/src/chain/channelmonitor.rs:1571]: `log_tx` adds "commitment tx" for us.

**Comment by TheBlueMatt** [comment]
Squashed, only change from the original PR is the minor typo:

```
-				log_info!(logger, "Broadcasting onchaing {}", log_tx!(bump_tx));
+				log_info!(logger, "Broadcasting onchain {}", log_tx!(bump_tx));
```

**Pull request merged**